lib/modules/hosts/imgur.js

```diff
@@ -14,7 +14,7 @@ addLibrary('mediaHosts', 'imgur', {
 
 	// hashRe: /^https?:\/\/(?:i\.|edge\.|www\.)*imgur\.com\/(?:r\/[\w]+\/)?([\w]{5,}(?:[&,][\w]{5,})?)(\..+)?(?:#(\d*))?$/i,
 	// the modified regex below fixes detection of "edited" imgur images, but imgur's edited images are broken right now actually, falling into
-	// a redirect loop.  preserving the old one just in case.  however it also fixes detection of the extension (.jpg, for example) which
+	// a redirect loop.	preserving the old one just in case.	however it also fixes detection of the extension (.jpg, for example) which
```

Comment:
Minor nit: whitespace. Not sure why there were double spaces there in the first place, you can replace them with single spaces or put them back, whatever you want.